src/services/auth/plugins/password/service.ts

```diff
@@ -193,7 +193,7 @@ export class MemberPasswordService {
     // Fetch the member's password
     const memberPassword = await memberPasswordRepository.getForMemberId(member.id);
     if (!memberPassword) {
-      throw new MemberWithoutPassword({ email });
+      throw new MemberWithoutPassword();
```

Comment:
On the function `getManyByEmail` from the `MemberRepository`, we're still throwing the `email`. Is it intended to have this difference ?

Reply:
I would say they are different, since in the `getManyByEmail` we are adding these errors to the `errors` key of the result object. But it is true we could omit it.